preview/Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM buildpack-deps:trusty-scm
 
+# Work around dotnet/cli#1582 until Docker releases a fix (docker/docker#20818).
```

Comment:
OK, I was wondering about that. I figured you did it intentionally.

I'll make that change to use the full URL. Other than that, does it look good now?